src/Integration/MefServices/ActiveSolutionBoundTracker.cs

```diff
@@ -60,11 +60,12 @@ internal sealed class ActiveSolutionBoundTracker : IActiveSolutionBoundTracker,
         public BindingConfiguration CurrentConfiguration { get; private set; }
 
         [ImportingConstructor]
-        public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolutionTracker, ILogger logger)
+        public ActiveSolutionBoundTracker(IHost host, IActiveSolutionTracker activeSolutionTracker, ILogger logger, IGitEvents gitEventsMonitor)
         {
             extensionHost = host ?? throw new ArgumentNullException(nameof(host));
             solutionTracker = activeSolutionTracker ?? throw new ArgumentNullException(nameof(activeSolutionTracker));
             this.logger = logger ?? throw new ArgumentNullException(nameof(logger));
+            if (gitEventsMonitor == null) { throw new ArgumentNullException(nameof(gitEventsMonitor)); }
```

Comment:
Checking for null parameters is pointless for MEF constructors (unless the constructor uses `ImportMany`, MEF just won't call the constructor with a `null` parameter.

I think we should just remove the null checks and unit tests in classes like this when we are making other changes to the file.